### PR TITLE
Create admin page including css, controller, and routes

### PR DESCRIPTION
**Pull request by JuriMino**
As a default, top page for admin shows all users

### CODE REVIEW

File: public/css/admin-style.css [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  font-family: "Font Awesome 5 Free";
+  font-weight: 900;
+  display: inline-block;
+  font-style: normal;
+  font-variant: normal;
+  text-rendering: auto;
+  }
+  .status-group{
+    display: inline;
+    width: 50%;
```
Comment (keanbillyfernandez) [inline public/css/admin-style.css:97]: why not put this in scss?

File: resources/views/admin/users.blade.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                            <td><i class="fa-solid fa-ellipsis"></i></td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+
+        </div>
+        <hr>
+        <div class="col footer pt-3 ps-3">
+            showing 1 to 10 of 50 users
```
Comment (keanbillyfernandez) [inline resources/views/admin/users.blade.php:107]: is this static in the end or just putting temporary here?

File: resources/views/admin/users.blade.php [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                            <td>Kei</td>
+                            <td>Kei Sasaki</td>
+                            <td>kei@gmail.com</td>
+                            <td>20xx-xx-xx</td>
+                            <td>20xx-xx-xx</td>
+                            <td><i class="fa-solid fa-circle text-success"></i></td>
+                            <td><i class="fa-solid fa-ellipsis"></i></td>
+                        </tr>
+                        <tr>
+                            <td>5</td>
```
Comment (keanbillyfernandez) [inline resources/views/admin/users.blade.php:90]: static? or just putting permanently?

File: resources/views/admin/users.blade.php [hunk 4]
```diff
@@ -0,0 +1,111 @@
+@extends('layouts.admin-app')
+
+@section('title','All Users')
+@section('content')
+<div class="row">
```
[the remark this answers is not in the document]
Comment (JuriMino) [reply resources/views/admin/users.blade.php:5]: container is included layouts.admin-app.blade.php which for extends.
it doesn't work?

File: resources/views/admin/users.blade.php (hunk 2 above)
Comment (JuriMino) [reply resources/views/admin/users.blade.php:107]: it's just temporary. I'm going to change later.

File: resources/views/admin/users.blade.php (hunk 3 above)
Comment (JuriMino) [reply resources/views/admin/users.blade.php:90]: it's just putting temporary.

**Pull request merged**